ui/page/send/page.go

```diff
@@ -40,12 +40,14 @@ type Page struct {
 	// helper methods for accessing the PageNavigator that displayed this page
 	// and the root WindowNavigator.
 	*app.GenericPageModal
+	modalLayout *cryptomaterial.Modal
```

Comment:
This isn't a modal layout, it's a modal. You should use it as `modal.Layout(..)`, not `modalLayout.Layout(..)`.
```suggestion
	modal *cryptomaterial.Modal
```

Reply:
this will conflict with  `"github.com/crypto-power/cryptopower/ui/modal"`

Reply:
It won't. This would be `pg.modal`, the other is just `modal`.